src/jobs/jobNodeActions.ts

```diff
@@ -0,0 +1,27 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as vscode from "vscode";
+import { DeleteJobs } from "@brightside/core";
+import * as nls from "vscode-nls";
+import { Job } from "../ZoweJobNode";
+const localize = nls.config({ messageFormat: nls.MessageFormat.file })();
+
+export async function deleteJob(node: Job) {
```

Comment:
Should the actions be part of the ZoweJobNode class?
(Just wondering what your thoughts are, no required change)

Reply:
Agree 100%.